docs/docs/manual/wikibase/overview.md

```diff
@@ -96,6 +96,24 @@ If your account or your bot is not properly authorized, OpenRefine will not disp
 
 You can store your unencrypted username and password in OpenRefine, saved locally to your computer and available for future use. For security reasons, you may wish to leave this box unchecked. You can also save your OpenRefine-specific bot password in your browser or with a password management tool. 
 
+Official manual of bot passwords: https://www.mediawiki.org/wiki/Manual:Bot_passwords
```

Comment:
I would rather avoid adding a URL in plain text like this (rather create a hyperlink with a meaningful label).